documentation/largescaledeployment.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Recommended configuration for large scale cluster deployment
+title: Validated scale limits for VCP
```

Comment:
On second thoughts, let's drop "validated".
New title -> Maximum scale supported by VCP.

Reply:
Addressed it latest commit!